code_generation/configuration.py

```diff
@@ -613,7 +618,7 @@ def _remove_empty_configkeys(self, config) -> None:
             Returns:
                 None
         """
-        for key in config:
+        for key in list(config):
```

Comment:
Why do we need to change it to a list?

Reply:
Removed in https://github.com/KIT-CMS/CROWN/pull/286/commits/c6acbae4d9494be64640dcec1c85f52fd76d5a4f